api/handlers/search/plate.go

```diff
@@ -0,0 +1,55 @@
+package search
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.uber.org/zap"
+
+	"github.com/linesmerrill/police-cad-api/config"
+	"github.com/linesmerrill/police-cad-api/databases"
+	"github.com/linesmerrill/police-cad-api/models"
+)
+
+type Plate struct {
+	DB                databases.VehicleDatabase
+	VerifyInCommunity func(string, string) bool
+}
+
+// Plate Search Handler ...
+func (p Plate) PlateSearchHandler(w http.ResponseWriter, r *http.Request) {
+	plateNumber := r.URL.Query().Get("plate")
+	communityID := r.URL.Query().Get("community_id")
+	userID := r.URL.Query().Get("user_id")
```

Comment:
suggestion: Yeah again, do we care what the `user_id` is when executing a plate search? No, because the person sending the query has no effect on the returned plate number.

```suggestion
```